backend/src/lib.rs

```diff
@@ -51,6 +52,8 @@ impl LTN {
     #[wasm_bindgen(constructor)]
     pub fn new(
         input_bytes: &[u8],
+        // Option doesn't work; the caller should just pass in 0 bytes to mean empty
```

Comment:
A doc comment would be good to remind us over in javascript land.

Reply:
`wasm.ts` wraps this to make a nicer TS API:

```
  constructor(
    osmInput: Uint8Array,
    demandInput: Uint8Array | undefined,
    boundary: Feature<Polygon>,
    studyAreaName: string | undefined,
  ) {
    this.inner = new LTN(
      osmInput,
      demandInput || new Uint8Array(),
      boundary,
      studyAreaName,
    );
  }
```